docassemble/HousingCodeChecklist/data/questions/customized_screens.yml

```
@@ -964,21 +964,22 @@ question: |
   What court does the tenant want to file in?
   % endif
 subquestion: |
+  % if len(all_matches) > 0:
+  Select an option from the list of court(s) serving the address you gave us,
+    % if isinstance(addresses_to_search, Iterable):
+    ${comma_and_list([address.on_one_line() for address in addresses_to_search],comma_string='; ')}.
+    % else:
+    ${addresses_to_search.on_one_line()}
+    % endif
+  % endif
+  
```

```suggestion
  % if len(all_matches) > 0:
  Select an option from the list of court(s) serving the address you gave us,
  % if isinstance(addresses_to_search, Iterable):
  ${comma_and_list([address.on_one_line() for address in addresses_to_search],comma_string='; ')}.
  % else:
  ${addresses_to_search.on_one_line()}
  % endif
  % endif
  
```
Spaces at the beginning of the line in Mako can change the formatting--once you reach 4 spaces, at least. So it's best not to use them even for nested conditional logic :(